### ISSUE TITLE
Shortened If Statement crashes Wist

### ISSUE THREAD

**Issue by IAmJoffa** [willowtreeapps/wist#75]
The shortened version of an 'If' statement causes Wist to crash.

Sample Code:
```brightscript
Sub Main(args As Dynamic)
    ' Check for Something
    if not myFunctionThatChecksSomething() return
End Sub
```

Call Stack Trace:
```
Cannot read property 'start' of undefined
TypeError: Cannot read property 'start' of undefined
    at checkThen (/Users/joffa/node_modules/@willowtreeapps/wist/lib/rules/no-shorthand-if.js:17:47)
    at Linter.ifSingleLineStatement:enter (/Users/joffa/node_modules/@willowtreeapps/wist/lib/rules/no-shorthand-if.js:25:17)
    at Linter.emit (events.js:182:13)
    at BrightScriptEventListener.enterIfSingleLineStatement (/Users/joffa/node_modules/@willowtreeapps/wist/parser/BrightScriptEventGenerator.js:84:22)
    at IfSingleLineStatementContext.enterRule (/Users/joffa/node_modules/@willowtreeapps/wist/parser/antlr/BrightScriptParser.js:4263:18)
    at ParseTreeWalker.enterRule (/Users/joffa/node_modules/antlr4/tree/Tree.js:207:6)
    at ParseTreeWalker.walk (/Users/joffa/node_modules/antlr4/tree/Tree.js:190:8)
    at ParseTreeWalker.walk (/Users/joffa/node_modules/antlr4/tree/Tree.js:193:9)
    at ParseTreeWalker.walk (/Users/joffa/node_modules/antlr4/tree/Tree.js:193:9)
    at ParseTreeWalker.walk (/Users/joffa/node_modules/antlr4/tree/Tree.js:193:9)
```

**Comment by nishtahir** [comment]
Thanks for reporting this. It looks like this is a bug in the rules engine.

**Comment by nishtahir** [comment]
Fixed in #76
